### PR TITLE
Make titles Bold in `nerdctl` output

### PR DESCRIPTION
**Pull request by aman556**
## Description
This PR is making the titles bold in the output of `nerdctl` command.

## Testing
- Build the project and run `nerdctl` command

## Output
Image in the first comment

Signed-off-by: Aman Sharma <amansh@vmware.com>

### CODE REVIEW

**Comment by aman556** [comment]
Screenshot 2022-12-12 at 11 27 10 PM
Notice that the Management commands:, Commands:, and Flags: are bold.

**Comment by Zheaoli** [comment]
Would you mind adding an image in the old version to let us know the effect detail?

**Comment by aman556** [comment]
@Zheaoli old version output image
Screenshot 2022-12-12 at 11 55 35 PM

**Review by Zheaoli — CHANGES_REQUESTED** [review]
LGTM for code

need squash the commit

**Comment by AkihiroSuda** [comment]
Thanks, but let me merge https://github.com/containerd/nerdctl/pull/1639 first

File: cmd/nerdctl/main.go [hunk 1]
```diff
@@ -43,6 +44,11 @@ const (
 	Management = "management"
 )
 
+var (
+	// To print Bold Text
+	Bold = color.New(color.Bold).SprintfFunc()
```
Comment (fahedouch) [inline cmd/nerdctl/main.go:49]: unexported var is fine :  `bold`

**Pull request merged**